src/extension.ts

```diff
@@ -51,6 +51,7 @@ import { logsKubernetes, LogsDisplayMode } from './components/kubectl/logs';
 import { Errorable, failed, succeeded } from './errorable';
 import { Git } from './components/git/git';
 import { DebugSession } from './debug/debugSession';
+import { suggestedShellForContainer } from './debug/debugUtils';
```

Comment:
This shouldn't be in `debugUtils` because it is used by non-debugging functions (specifically the `Terminal` command I think).  Consider a new module in the `utils` folder e.g. `containerutils.ts` or even `container-shell.ts` if you want to be specific.